### PR TITLE
add absolute value support

### PR DESCRIPTION
**Pull request by bendavies**
Adds a method to `Money` and calculators for returning the absolute value.

### CODE REVIEW

**Comment by frederikbosch** [comment]
@bendavies Are you abs for money or other purposes? I have the feeling that you like the classes to be used outside `Money`, right?

**Comment by frederikbosch** [comment]
Oh sorry, I did not see the absolute method on `Money`. It actually looks quite good.

**Comment by frederikbosch** [comment]
@sagikazarmark I think we can merge this one in.

**Comment by bendavies** [comment]
note that i've called it `abs` on the calculators and `absolute` on money.
which do you prefer? shall we name them the same?

File: spec/Calculator/CalculatorBehavior.php [hunk 1]
```diff
@@ -62,6 +62,12 @@ function it_floors_a_value()
         $this->floor(-2.7)->shouldReturn('-3');
     }
 
+    function it_absolutes_a_value()
```
Comment (sagikazarmark) [inline spec/Calculator/CalculatorBehavior.php]: it_calculates_the_absolute_value?

File: spec/MoneySpec.php [hunk 2]
```diff
@@ -388,4 +388,32 @@ function comparatorExamples()
             ['-1', false, false, true],
         ];
     }
+
+
+    /**
+     * @dataProvider absoluteExamples
+     */
+    function it_absolutes_the_amount($amount, $result, Calculator $calculator)
```
Comment (sagikazarmark) [inline spec/MoneySpec.php]: same here

**Comment by sagikazarmark** [comment]
Two small comments, LGTM apart from those.

**Comment by frederikbosch** [comment]
@bendavies I rather name the methods the same: `absolute` for `Money` and `Calculator` implementations.

**Comment by bendavies** [comment]
comments addressed. 
@sagikazarmark went for `it_calculates_the_absolute_amount/value` which follows the convention of the existing specs on the class.

**Pull request merged**